src/backends/perf_pt/collect.c

```diff
@@ -232,7 +339,22 @@ poll_loop(int perf_fd, int stop_fd, struct perf_event_mmap_page *mmap_hdr,
         // POLLIN on pfds[0]: Overflow event on either the Perf AUX or data buffer.
         // POLLHUP on pfds[1]: Tracer stopped by parent.
         if ((pfds[0].revents & POLLIN) || (pfds[1].revents & POLLHUP)) {
-            read_aux(aux, mmap_hdr, trace, err);
+            // Read from the Perf file descriptor.
+            // We don't actually use any of what we read, but it's probably
```

Comment:
Probably?

Reply:
It's a gut feeling. No one says I should empty the fd, but something tells me we can't buffer it up forever. Don't you agree?

Reply:
I guess I also partly wondered why we're throwing everything away...

Reply:
You only read counters out of the fd, which we don't need for our use case.